ui/main/src/app/modules/admin/components/table/admin-table.directive.ts

```diff
@@ -216,6 +216,11 @@ export abstract class AdminTableDirective implements OnInit, OnDestroy {
     return this.translateService.instant(this.i18NPrefix + headerIdentifier);
   }
 
+  onFilterChanged(event) {
+    this.page = 1;
+    this.gridApi.paginationGoToPage(this.page-1);
```

Comment:
why not this.gridApi.paginationGoToPage(**0**) ?

Reply:
True. Done.